llvm/include/llvm/CodeGen/MachineLoopInfo.h

```diff
@@ -66,6 +67,14 @@ class MachineLoop : public LoopBase<MachineBasicBlock, MachineLoop> {
   /// it returns an unknown location.
   DebugLoc getStartLoc() const;
 
+  /// \brief Find the llvm.loop metadata for this loop.
+  /// If each branch to the header of this loop contains the same llvm.loop
```

Comment:
Should we add a blank line after line 70?
https://llvm.org/docs/CodingStandards.html

> The first sentence (or a passage beginning with \brief) is used as an abstract. Any additional information should be separated by a blank line. If an algorithm is based on a paper or is described in another source, provide a reference.

Reply:
Will remove `\brief`.

Reply:
830699979b3d24edd97ddbb389f348491106776c